pkg/build/client/clients.go

```diff
@@ -0,0 +1,69 @@
+package client
```

Comment:
This is more of a convention thing.  If you have one file in a package, and you want to call the package a certain something, you generally call the only file in the package the same as the package.  That's usually the stronger convention.